hypothesis-ruby/Rakefile

```diff
@@ -116,12 +116,14 @@ task :gem do
 
   previous_version = spec.version.to_s
 
-  if run_for_output('git', 'tag', '-l', previous_version).empty?
+  previous_tag = "hypothesis-ruby-#{previous_version}"
+
+  if run_for_output('git', 'tag', '-l', previous_tag).empty?
```

Comment:
nit: is this `-l`, short for `--list`?

If so, longer flag would be preferred, both for readability and to avoid confusion with badly designed monospaced fonts.

Reply:
In actual fact this change didn't get used and this code needs some heavy reworking, so I'm just going to revert it to the version in master, but agreed RE preference for `--list` over `-l`.